blueprint/src/loops.tex

```diff
@@ -222,9 +222,9 @@ \subsection{Surrounding families}
 The difficulty in constructing global families of surrounding loops is that
 there are plenty of surrounding loops and we need to choose them
 consistently.
-The key feature of the above definition is that the $t$ parameter,
-in addition to allowing to cut out the corrugation
-process in the next chapter, brings a satisfied or refund guarantee,
+The key feature of the above definition is that the $t$ parameter not only
+allows us to cut out the corrugation
+process in the next chapter, but also brings a "satisfied or refund" guarantee,
```

Comment:
```suggestion
process in the next chapter, but also brings a ``satisfied or refund'' guarantee,
```

Reply:
Thanks, I forgot about that! It's been too long since I had to write LaTeX...